src/Token/AccessToken.php

```diff
@@ -76,6 +76,9 @@ public function __construct(array $options = [])
         // We need to know when the token expires. Show preference to
         // 'expires_in' since it is defined in RFC6749 Section 5.1.
         // Defer to 'expires' if it is provided instead.
+        // Initialize 'expires' to false, meaning that it is not known
+        // or doesn't expire
+        $this->expires = false;
```

Comment:
No reason in particular. I think it makes sense that expires is false means that it doesn't expire... but could also be null as in it is not defined.